### ISSUE TITLE
Available strategies

### ISSUE THREAD

**Issue by boldt** [jurassix/docs-react-validation-mixin#1]
It would be nice to list all available strategies within the docs.

**Comment by jurassix** [comment]
Agreed. Right now I only know of two strategies: [joi-validation-strategy](https://github.com/jurassix/joi-validation-strategy), [yup-validation-strategy](https://github.com/wali-s/yup-validation-strategy)

I was thinking about a `package.json` keyword, _validation-strategy_, to help automate this; but I need more time for discovery.

I'm currently working on the yup-validation-strategy, waiting on the owner to migrate tests. Once that is complete the docs will be updated.
